charts/kyuubi/templates/_helpers.tpl

```diff
@@ -31,3 +31,21 @@ For details, see 'kyuubi.frontend.protocols': https://kyuubi.readthedocs.io/en/m
   {{- end }}
   {{- $protocols |  join "," }}
 {{- end }}
+
+{{/*
+Generate kyuubi common labels.
```

Comment:
the code is short and clear, the comment should explain the purpose of the definition instead of what the code does.

Labels used by selectors to match Kyuubi resources.

Reply:
ok, change the labels comment